cms/djangoapps/contentstore/rest_api/v1/views/tests/test_course_index.py

```diff
@@ -150,6 +150,6 @@ def test_number_of_calls_to_db(self):
         """
         Test to check number of queries made to mysql and mongo
         """
-        with self.assertNumQueries(29, table_ignorelist=WAFFLE_TABLES):
+        with self.assertNumQueries(31, table_ignorelist=WAFFLE_TABLES):
```

Comment:
These are from the tag count queries

Reply:
Now +3: (1 for the course as a whole, 1 for all the blocks in the course, and 1 for getting the org-specific value of `COURSE_AUTHORING_MICROFRONTEND_URL` (within `get_course_authoring_url` within `get_taxonomy_tags_widget_url` within `_get_course_index_context`).